diff --git a/vendor/bower-components/tag-it/js/tag-it.js b/vendor/bower-components/tag-it/js/tag-it.js
index 3adcf4c..1c451ea 100644
--- a/vendor/bower-components/tag-it/js/tag-it.js
+++ b/vendor/bower-components/tag-it/js/tag-it.js
@@ -241,7 +241,6 @@
                     // Tab will also create a tag, unless the tag input is empty,
                     // in which case it isn't caught.
                     if (
-                        (event.which === $.ui.keyCode.COMMA && event.shiftKey === false) ||
                         event.which === $.ui.keyCode.ENTER ||
                         (
                             event.which == $.ui.keyCode.TAB &&
@@ -271,6 +270,15 @@
                             that.createTag(that._cleanedInput());
                         }
                     }
+                }).keypress(function(event){
+                    if (String.fromCharCode(event.which) === that.options.singleFieldDelimiter) {
+                        event.preventDefault();
+                        // Autocomplete will create its own tag from a selection and close automatically.
+                        if (!(that.options.autocomplete.autoFocus && that.tagInput.data('autocomplete-open'))) {
+                            that.tagInput.autocomplete('close');
+                            that.createTag(that._cleanedInput());
+                        }
+                    }
                 }).blur(function(e){
                     // Create a tag when the element loses focus.
                     // If autocomplete is enabled and suggestion was clicked, don't add it.
